src/app.js

```diff
@@ -1,21 +1,40 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
+const PORT = process.env.PORT || 8080;
+
+const server = http.createServer((req, res) => {
+  const url = new URL(req.url, `http://${req.headers.host}`);
+
+  if (!(url.pathname.startsWith('/file') || req.url === '/')) {
+    res.end('the path should start with "/file/"');
+
+    return;
+  }
+
+  const fileName = url.pathname.slice(6) || 'index.html';
+  const filePath = path.resolve('public', fileName);
+
+  if (!fs.existsSync(filePath)) {
+    res.statusCode = 404;
+    res.end('File does not exist');
+
+    return;
+  }
+
+  fs.readFile(filePath, 'utf-8', (err, data) => {
+    if (err) {
+      res.statusCode = 500;
+      res.end('Something went wrong');
```

Comment:
`Something went wrong` is not the best error description, but it depends on what we want to show for user
Generally we can send err message, or somehow write custom error to provide more descriptive information about error